DEVELOPMENT.md

```diff
@@ -0,0 +1,185 @@
+    This will further solidify k8s nomenclature
+-   [API conventions - Types(kinds)](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#types-kinds) -
+    Another useful set of words describing words. "Objects" and "Lists" in k8s
+    land
+-   [Extend the Kubernetes API with CustomResourceDefinitions](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/)-
+    A tutorial demonstrating how a Custom Resource Definition can be added to
+    Kubernetes without anything actually "happening" beyond being able to list
+    Objects of that kind
+
+At this point, you may find it useful to return to these `Shipwright Build` docs:
+
+-   [Shipwright Build](https://github.com/shipwright-io/build/blob/master/README.md) -
+    Some of the terms here may make more sense!
+-   Install via [getting started for development](#getting-started)
+-   [Shipwright Build overview and tutorial](https://github.com/shipwright-io/build/blob/master/docs/README.md) -
+    Define `BuildStrategies`, `Builds`, and `BuildRuns`, see what happens when
+    they are run
+
+### Checkout your fork
+
+The Go tools require that you clone the repository to the
+`src/github.com/shipwright-io/build` directory in your
+[`GOPATH`](https://github.com/golang/go/wiki/SettingGOPATH).
+
+To check out this repository:
+
+1.  Create your own
+    [fork of this repo](https://help.github.com/articles/fork-a-repo/)
+1.  Clone it to your machine:
+
+```shell
+mkdir -p ${GOPATH}/src/github.com/shipwright-io
+cd ${GOPATH}/src/github.com/shipwright-io
+git clone git@github.com:${YOUR_GITHUB_USERNAME}/build.git
+cd build
+git remote add upstream git@github.com:shipwright-io/build.git
+git remote set-url --push upstream no_push
+```
+
+_Adding the `upstream` remote sets you up nicely for regularly
+[syncing your fork](https://help.github.com/articles/syncing-a-fork/)._
+
+### Requirements
+
+You must install these tools:
+
+1.  [`go`](https://golang.org/doc/install): The language Shipwright Build is
+    built in
+1.  [`git`](https://help.github.com/articles/set-up-git/): For source control
+1.  A container runtime to build the operator image, such as [`docker`](https://docs.docker.com/get-docker/) or [podman](https://podman.io/)
+1.  [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/): For
+    interacting with your kube cluster
+
+### Create a cluster and a repo
+
+1. [Set up a kubernetes cluster](https://kubernetes.io/docs/setup/)
+   - Follow the instructions in the Kubernetes doc.
+1. Set up a docker repository for pushing images. You can use any container
+   image registry by adjusting the authentication methods and repository paths
+   mentioned in the sections below.
+   - [Docker Hub quickstart](https://docs.docker.com/docker-hub/)
+   - [IBM Container Registry quickstart](https://cloud.ibm.com/docs/Registry)
```

Comment:
I would try to be a bit more neutral here:

```suggestion
1. Set up a container image repository for pushing images. Any container image registry that is accessible to your cluster can be used for your repository. This can be a public registry like [Docker Hub])(https://docs.docker.com/docker-hub/), [quay.io](https://quay.io), or a container registry run by your cloud provider.
```

Reply:
Sure thing. Done